### DISCUSSION TITLE
Copy model specified material error

### DISCUSSION THREAD

**Discussion by anan1213095357** [stride3d/stride#2447] [Q&A]
version last
system win10

When I copied the model, everything was normal and smooth, but if I added new materials to the copied model, an error would occur.

![image](https://github.com/user-attachments/assets/5a9aaec5-8daf-44ca-afeb-587b20759abd)
![image](https://github.com/user-attachments/assets/fce640ef-b8c8-42c0-abb7-bce572937c21)
![image](https://github.com/user-attachments/assets/7e5f5a32-33a8-45d5-82f9-2fd4d7f4c7a8)
![image](https://github.com/user-attachments/assets/41b0063a-2696-4118-9f49-10b078ca8ff1)

**Comment by Kryptos-FR** [comment]
@anan1213095357 Thanks for coming here. GitHub Issues are meant to be used for actual bugs that are reproducible and likely a problem in the engine.

It looks like your problem is more about knowing on how to use the engine, in which case I would suggest to use the discussions panel or to join the discord community and ask your questions there, so that this GitHub Issues space doesn't get overwhelmed.

I will convert it into a discussion as well as #2442.
